src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  NavLink,
+} from 'react-router-dom';
+import Home from './views/Home/Home';
+import './App.css';
+import TeamDetail from './views/Teams/TeamDetail.jsx';
+import Teams from './views/Teams/Teams.jsx';
+import Players from './views/Players/Players.jsx';
+import PlayerDetail from './views/Players/PlayerDetail.jsx';
+
+function App() {
+  return (
+    <main className="App-header">
+      <Router>
+        <header>
+          <NavLink to="/">Home</NavLink> <NavLink to="/teams">Teams</NavLink>{' '}
```

Comment:
another way to space these out in css instead of using `{' '}` would look something like 
`header a {
     margin: 1rem
}`